modules/seo.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Module Name: seo
+ * Module Description: Adds tools to enhance your site's content for better results on search engines and social media.
+ * Sort Order: 35
+ * First Introduced: 4.3
+ * Requires Connection: No
+ * Auto Activate: Yes
+ * Module Tags: Social, Appearance
+ */
+
+include dirname( __FILE__ ) . '/seo/helpers-seo.php';
+include dirname( __FILE__ ) . '/seo/title-seo.php';
+include dirname( __FILE__ ) . '/seo/posts-seo.php';
+
+/**
+ * An SEO expert walks into a bar, bars, pub, public house, Irish pub, drinks, beer, wine, liquor, Grey Goose, Cristal...
+ */
+class SEO {
+	function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	function init() {
+		add_action( 'wp_head', array( $this, 'meta_tags' ) );
```

Comment:
@jeherve I've refactored the existing code that modifies OG tags in order to use them.
As for non-OG meta tags, I'm wondering if we should plug them there, since all of the OG tags have `<meta property="%s" content="%s" />` format, and SEO tags use `<meta name="%s" content="%s" />`. What do you think?

If the primary reason behind that is plugin compatibility, maybe we could use `jetpack_enable_open_graph` here to check whether other SEO meta tags should be enabled?